lib/reflectable_builder.dart

```diff
@@ -50,12 +50,29 @@ Future<BuildResult> reflectableBuild(List<String> arguments) async {
   } else {
     // TODO(eernst) feature: We should support some customization of
     // the settings, e.g., specifying options like `suppress_warnings`.
-    BuilderOptions options = new BuilderOptions(
+    var options = new BuilderOptions(
         <String, dynamic>{"entry_points": arguments, "formatted": true},
         isRoot: true);
     final builder = new ReflectableBuilder(options);
-    return await build(
-        [applyToRoot(builder, generateFor: new InputSet(include: arguments))],
-        deleteFilesByDefault: true);
+    List<BuilderApplication> builders = [
+      applyToRoot(builder, generateFor: new InputSet(include: arguments))
```

Comment:
Note that if the input set here changes, that changes the structure of the build, which will cause us to throw away any old work and rebuild the world.

If you modify the builder options instead (which it looks like you already do) then we will only invalidate the builders that use those options.

In this case though, I don't think it really makes a difference since there is only a single builder.

Reply:
OK, then I think I'll leave it as it is for now. Thanks!